<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulateWidget</class>
 <widget class="QWidget" name="SimulateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0" columnstretch="1,0,0">
   <item row="3" column="2">
    <widget class="QPushButton" name="fitButton">
     <property name="text">
      <string>Fit By Simulation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>151</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>699</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="plotWidget" native="true"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QTableView" name="parametersWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
